### DISCUSSION TITLE
2020 年全球前十大 SSD 模組廠品牌排名出爐，整體出貨量年減 15%

### DISCUSSION THREAD

**Discussion by 239216** [1101-GS4719-C/Python-Community-GS4719-C#130] [General]
根據 TrendForce 研究顯示，由於新冠肺炎疫情造成生產供應及物流延遲，尤其在 2020 年第二季起，全球普遍採取封閉管理措施加以因應，導致訂單量急速衰退。
疫情衝擊加速汰除二、三線供應商，前十大品牌出貨量占比再擴大
長江存儲崛起加大國產化力道，品牌優勢逐步轉向中資企業
PCIe G4推出成各家新產品主力，QLC加入模組廠供應行列
以出貨市占排名來看，前三大品牌仍是金士頓（Kingston）、威剛（ADATA）及金泰克（Kimtigo）。
https://technews.tw/2021/10/25/ssd-major-on-third-brand/

**Comment by HSU-YU-JEN** [comment]
沒想到疫情竟然也影響到SSD的出貨!!!

**Comment by Isongzhe** [comment]
感覺最近的價格有降回來。之前我才因為一批G4的硬碟出貨，直接搶到一條便宜的1T固態
